src/Adapter/Product/AdminProductDataProvider.php

```diff
@@ -338,7 +338,7 @@ public function getCatalogProductList(
         $sqlWhere[] = 'state = ' . Product::STATE_SAVED;
 
         // exec legacy hook but with different parameters (retro-compat < 1.7 is broken here)
-        Hook::exec('actionAdminProductsListingFieldsModifier', [
+        $this->hookDispatcher->dispatchWithParameters('actionAdminProductsListingFieldsModifier', [
```

Comment:
According to PHPStan you are using an undefined property

Reply:
fixed. excuseme